pkg/test/verify/cluster.go

```diff
@@ -47,21 +47,9 @@ func AddToolchainClusterAsMember(t *testing.T, functionToVerify FunctionToVerify
 				require.NoError(t, err)
 				cachedToolchainCluster, ok := cluster.GetCachedToolchainCluster("east")
 				require.True(t, ok)
-				assert.Equal(t, cluster.Member, cachedToolchainCluster.Type)
-				if labels["namespace"] == "" {
-					assert.Equal(t, "toolchain-member-operator", cachedToolchainCluster.OperatorNamespace)
-				} else {
-					assert.Equal(t, labels["namespace"], cachedToolchainCluster.OperatorNamespace)
-				}
+
 				// check that toolchain cluster role label tenant was set only on member cluster type
 				require.NoError(t, cl.Get(context.TODO(), client.ObjectKeyFromObject(toolchainCluster), toolchainCluster))
-				expectedToolChainClusterRoleLabel := cluster.RoleLabel(cluster.Tenant)
-				_, found := toolchainCluster.Labels[expectedToolChainClusterRoleLabel]
-				if labels["type"] == string(cluster.Member) {
-					require.True(t, found)
-				} else {
-					require.False(t, found)
-				}
```

Comment:
I think we still need to check that the `cluster.Tenant` role is being set on the toolchaincluster CRs that are in toolchain-host-operator ( thus representing member clusters ). Don't we ?

Reply:
Yup, I agree with @mfrancisc. But let's expect that the label is properly set in `cachedToolchainCluster.Labels["namespace"]`, we can assume that the add-cluster.sh script does that.